widgets/views/comments.php

```diff
@@ -48,6 +44,10 @@
                                         ?>
                                     </div>
                                 </div>
+                                <div class="comment-footer">
+                                    <a href="#"><?= $comment->user->username; ?></a> at
```

Comment:
same as it was above, this should be `<?= $comment->user ? $comment->user->rankLink : User::DELETED_USER_HTML ?>`